README.md

```diff
@@ -1,50 +1,63 @@
 # Using the Learn IDE (Integrated Development Environment)
 
-The Learn Integrated Development Environment (IDE) is what you'll be using to write code and submit your labs on Learn. Let's take a closer look at what it is and how it works. 
+You are probably familiar with using an application like Microsoft Office, Open Office, or Google Docs to create things like documents and spreadsheets. You've probably also used applications like Safari, Chrome, or Internet Explorer that allow you to access and interact with the internet. What kind of program allows us to write code?
 
-We created the Learn IDE based off of GitHub's very popular text editor, Atom. We've taken advantage of its highly configurable design by optimizing it to solve labs on Learn in two key ways. (1) We've configured a server in the cloud with all the tools and packages you need to build software, as well as the files you'll work on while solving Learn Labs. (2) We've given you access to that server via a Terminal window within the app.
+There are many different applications that allow you to both see and edit code. These are called text editors. One advantage of some text editors is they allow you to customize them in a highly personalized manner. There are different plug-ins that you can download to allow you to do everything from simply changing text style and color to auto-formatting and auto-complete. Some text editors you might have heard of include Vi, Atom, Sublime, Notepad, and Visual Studio Code.
+
+The Learn Integrated Development Environment (IDE) is what you'll be using to write code and submit your labs on Learn. We created the Learn IDE based off of GitHub's very popular text editor, Atom. We've taken advantage of its highly configurable design by optimizing it to solve labs on Learn in two key ways. (1) We've configured a server in the cloud with all the tools and packages you need to build software, as well as the files you'll work on while solving Learn Labs. (2) We've given you access to that server via a Terminal window within the app.
```

Comment:
I think the ordering is slightly off here since you go text editor --> learn. Maybe something like...

> The Learn Integrated Development Environment (IDE) is a web-based version of GitHub's text editor, Atom, _plus_ a (terminal / shell / command prompt) &mdash; _all in one_. When you launch Learn Learn IDE you'll see a "file tree" and a text editing space. Those are provided by Atom. The terminal at the bottom is something we've added to help you test and interact with your code. We'll be giving you a tour in the rest of this lesson.

I think some of the subtleties of point `1` are why we think Learn is spiffy, but it might not translate to learners at this point in time.